example/lib/spring_page.dart

```diff
@@ -22,9 +22,12 @@ class _SpringPageState extends State<SpringPage> with TickerProviderStateMixin {
 
   // Get the current spring description
   SpringDescription get _currentSpring => switch (_springType) {
-        SpringType.defaultIOS => Spring.defaultIOS,
-        SpringType.bouncy => Spring.bouncy,
-        SpringType.snappy => Spring.snappy,
+        SpringType.defaultIOS =>
+          (Motion.smoothSpring() as SpringMotion).description,
+        SpringType.bouncy =>
+          (Motion.bouncySpring() as SpringMotion).description,
+        SpringType.snappy =>
+          (Motion.snappySpring() as SpringMotion).description,
```

Comment:
You can probably avoid the type casting like this:

```suggestion
         CupertinoMotion.smooth().description,
        SpringType.bouncy =>
          CupertinoMotion.bouncy().description,
        SpringType.snappy =>
          CupertinoMotion.snappy().description,
```

Reply:
oops, I didn't find this! Thanks!

Reply:
fixed 1bd0a2bd095c6cdfcc48305c282b38a998d308df